cuml_bench/elasticnet.py

```diff
@@ -39,7 +39,7 @@
 X_train, X_test, y_train, y_test = bench.load_data(params)
 
 # Create our regression object
-regr = ElasticNet(fit_intercept=params.fit_intercept, l1_ratio=params.l1_ratio,
+regr = ElasticNet(fit_intercept=False, l1_ratio=params.l1_ratio,

```

Comment:
set 'fit_intercept' to False since default value is changed in new sklearn version